Formula/mongodb-mongocryptd.rb

```diff
@@ -0,0 +1,18 @@
+class MongodbMongocryptd < Formula
+  desc "mongocryptd service for Client Side Encryption"
+  homepage "https://www.mongodb.com/"
+
+  url "https://downloads.mongodb.com/osx/mongodb-cryptd-macos-x86_64-enterprise-4.4.6.tgz"
+  sha256 "bdf8aa2110b1f72c847b6b49f73fb855ae58549a2640ea69d7971740d7ff10db"
+  license "MongoDB Customer Agreement"
+
+  def caveats
+    <<~EOS
+      mongocryptd is licensed under the MongoDB Customer Agreement (https://www.mongodb.com/customer-agreement). Except for evaluation purposes, you may not use mongocryptd without a commercial license from MongoDB.
+    EOS
+  end
+
+  test do
+    system "#{bin}/mongocryptd", "--version"
```

Comment:
The cryptd package above contains a copy of `LICENSE-Enterprise.txt` which the community server package does not. Should we be installing that as well, per the caveats above?

Reply:
Stupid question maybe but - where would I install the `LICENSE-Enterprise.txt` file too? 🤔

Reply:
To the root, where you would find the `LICENSE-Community.txt` from a community install.

Reply:
I'm now using the same `prefix.install` as the community install does so it will be located in the Cellar's folder for `mongodb-mongocryptd` 👍